### DISCUSSION TITLE
Changing the name of the repository?

### DISCUSSION THREAD

**Discussion by RudolphRiedel** [RudolphRiedel/FT800-FT813#19] [General]
I am thinking about renaming the repository as it is obviously outdated by now.
Actually I am thinking about renaming it for some time now but the downside is that this will break every single link in every browser that has been set to this repository.

The new name could be:
EVE_Embedded_Video_Engine

**Comment by dstulken** [comment]
If you do rename it, you might want to choose something a bit more unique so that it is searchable?

For example, a Google search for "eve embedded video engine" already gives 1.4 million results.  It's going to be hard for your work to be found, even by someone who knows the name, because anyone else who writes code for these chips or does an EVE-based project is going to have the same keywords in their project descriptions...

To anyone I've told about this project, I've always called it the "Riedel EVE display library".   It's not the official name of course, but it *is* searchable!

**Reply by RudolphRiedel** [reply]
That is annother good argument against changing the name, Google does find the repository with a couple of different searches now and this may break this a little.
Well, things evolve and good names are difficult.
This is why I am using names like "L-D5019-05-03" for my PCB projects. :-)
